contracts/DelFiPrice.sol

```diff
@@ -58,7 +128,8 @@ contract DelFiPrice is OpenOracleView {
         for (uint i = 0; i < symbols.length; i++) {
             string memory symbol = symbols[i];
             uint64 medianPrice = medianPrice(symbol, sources);
-            uint64 anchorPrice = OpenOraclePriceData(address(data)).getPrice(anchor, symbol);
+            // uint64 anchorPrice = OpenOraclePriceData(address(data)).getPrice(anchor, symbol);
+            uint64 anchorPrice = uint64(AnchorPriceOracle(address(anchor)).getUnderlyingPrice(getCTokenAddress(symbol)));
```

Comment:
Maybe break this into multiple steps? It's doing.. a lot for one 1 LoC

Reply:
I will do after merging Coburn's pr, so that we have the final version of this line before splitting it.